src/Gemfile

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
+gem "puma"
```

Comment:
🤔 Wellll, since we don't actually publish the gem to RubyGems.org, the dependency doesn't _have_ to be declared in the gemspec. And the Gemfile is included in the packaging and for the build's `bundler install`, soooo ... yea. That'd work, too.